cluster/gce/gci/master.yaml

```diff
@@ -74,6 +74,33 @@ write_files:
       [Install]
       WantedBy=kubernetes.target
 
+  - path: /etc/systemd/system/kube-logrotate.timer
```

Comment:
Nop. This does not generate any logs. This is simply a timer. Neither does kube-logrotate.service.
All active timers can be seen by running "systemctl list-timers".